src/EventHandlers/Pool.ts

```diff
@@ -115,13 +134,17 @@ Pool.Fees.handlerWithLoader({
 Pool.Swap.handlerWithLoader({
   loader: async ({ event, context }) => {
     const liquidityPoolNew = await context.LiquidityPoolNew.get(
-      event.srcAddress.toString()
+      toChecksumAddress(event.srcAddress)
```

Comment:
Believe this will be already returned as checksummed so probably no need to checksum, but doesn't affect anything